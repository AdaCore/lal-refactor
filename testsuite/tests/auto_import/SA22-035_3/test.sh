#!/bin/sh
auto_import -P default.gpr --source ./src/a.adb --line 46 --column 20
