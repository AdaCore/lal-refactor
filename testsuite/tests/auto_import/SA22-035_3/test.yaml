description: This test checks that the tool does not suggest anything if the name is resolved.

driver: shell_script
