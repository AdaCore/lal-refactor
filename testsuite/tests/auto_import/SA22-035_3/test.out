Available Imports:
Import: A; Qualifier: A.B.C.D.IHM
Import: A; Qualifier: A.B.C.D.Integer_Hashed_Maps
Import: A; Qualifier: A.B.C.D.PIHM
Import: A; Qualifier: A.B.C.D.Private_Integer_Hashed_Maps
Import: A; Qualifier: A.B.C.IHM
Import: A; Qualifier: A.B.C.Integer_Hashed_Maps
Import: A; Qualifier: A.B.C.PIHM
Import: A; Qualifier: A.B.C.Private_Integer_Hashed_Maps
Import: A; Qualifier: A.B.IHM
Import: A; Qualifier: A.B.Integer_Hashed_Maps
Import: A; Qualifier: A.B.PIHM
Import: A; Qualifier: A.B.Private_Integer_Hashed_Maps
Import: A; Qualifier: A.IHM
Import: A; Qualifier: A.Integer_Hashed_Maps
Import: A; Qualifier: A.PIHM
Import: A; Qualifier: A.Private_Integer_Hashed_Maps
Import: E; Qualifier: E.B.C.D.IHM
Import: E; Qualifier: E.B.C.D.Integer_Hashed_Maps
Import: E; Qualifier: E.B.C.D.PIHM
Import: E; Qualifier: E.B.C.D.Private_Integer_Hashed_Maps
Import: E; Qualifier: E.B.C.IHM
Import: E; Qualifier: E.B.C.Integer_Hashed_Maps
Import: E; Qualifier: E.B.C.PIHM
Import: E; Qualifier: E.B.C.Private_Integer_Hashed_Maps
Import: E; Qualifier: E.B.IHM
Import: E; Qualifier: E.B.Integer_Hashed_Maps
Import: E; Qualifier: E.B.PIHM
Import: E; Qualifier: E.B.Private_Integer_Hashed_Maps
Import: E; Qualifier: E.IHM
Import: E; Qualifier: E.Integer_Hashed_Maps
Import: E; Qualifier: E.PIHM
Import: E; Qualifier: E.Private_Integer_Hashed_Maps
Import: F; Qualifier: F.B.C.D.IHM
Import: F; Qualifier: F.B.C.D.Integer_Hashed_Maps
Import: F; Qualifier: F.B.C.D.PIHM
Import: F; Qualifier: F.B.C.D.Private_Integer_Hashed_Maps
Import: F; Qualifier: F.B.C.IHM
Import: F; Qualifier: F.B.C.Integer_Hashed_Maps
Import: F; Qualifier: F.B.C.PIHM
Import: F; Qualifier: F.B.C.Private_Integer_Hashed_Maps
Import: F; Qualifier: F.B.IHM
Import: F; Qualifier: F.B.Integer_Hashed_Maps
Import: F; Qualifier: F.B.PIHM
Import: F; Qualifier: F.B.Private_Integer_Hashed_Maps
Import: F; Qualifier: F.IHM
Import: F; Qualifier: F.Integer_Hashed_Maps
Import: F; Qualifier: F.PIHM
Import: F; Qualifier: F.Private_Integer_Hashed_Maps

Edits:
a.adb
1:1-1:1 with A;

46:20-46:20 A.B.C.D.IHM.

